server/config.py

```diff
@@ -37,8 +37,6 @@
     )
     sys.exit(1)
 
-bind = "0.0.0.0:5000"
```

Comment:
We can leave it there. It's just default value.

Reply:
We actually removed this on purpose, so the user can define it at runtime. In some use-cases this has practical advantage, and indeed, as this is the default value I don't see why to hardcode it in the config.